addons/account/views/account_invoice_view.xml

```diff
@@ -47,18 +47,30 @@
                     <field name="uom_id" groups="uom.group_uom"/>
                     <field name="price_unit"/>
                     <field name="discount" groups="base.group_no_one"/>
-                    <field name="price_subtotal"/>
+                    <field name="price_subtotal" groups="account.group_show_price_subtotal"/>
+                    <field name="price_total" groups="account.group_show_price_total"/>
                     <field name="currency_id" invisible="1"/>
                 </tree>
             </field>
         </record>
 
+        <!--
+            NOTE: This view uses "parent" and will not work standalone.
+            But for now we don't need it standalone.
+        -->
         <record id="view_invoice_line_form" model="ir.ui.view">
             <field name="name">account.invoice.line.form</field>
             <field name="model">account.invoice.line</field>
             <field name="arch" type="xml">
-                <form string="Invoice Line">
-                    <group>
+                <form>
+                    <!--
+                        We need the sequence field to be here
+                        because we want to be able to overwrite the default sequence value in the JS
```

Comment:
You can be more clear/concise in your comments ;)

Reply:
Do you have an example of what would be better in this case?